### ISSUE TITLE
java.util.ServiceConfigurationError: org.slf4j.spi.SLF4JServiceProvider: org.slf4j.jul.JULServiceProvider not a subtype at java.base/java.util.ServiceLoader.fail(ServiceLoader.java:593)

### ISSUE THREAD

**Issue by draconnn** [JetBrains/spotbugs-intellij-plugin#1522]
Product Infos
    SpotBugs: 4.4.2
    IntelliJ SpotBugs plugin: 1.2.5
    IDEA: IntelliJ IDEA 2023.3
    IDEA-Build: IU-233.11799.241, December 2, 2023
    JRE: 17.0.9+7-b1087.7 amd64
    JVM: OpenJDK 64-Bit Server VM JetBrains s.r.o.

null
java.util.ServiceConfigurationError: org.slf4j.spi.SLF4JServiceProvider: org.slf4j.jul.JULServiceProvider not a subtype
	at java.base/java.util.ServiceLoader.fail(ServiceLoader.java:593)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNextService(ServiceLoader.java:1244)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNext(ServiceLoader.java:1273)
	at java.base/java.util.ServiceLoader$2.hasNext(ServiceLoader.java:1309)
	at java.base/java.util.ServiceLoader$3.hasNext(ServiceLoader.java:1393)
	at org.slf4j.LoggerFactory.findServiceProviders(LoggerFactory.java:106)
	at org.slf4j.LoggerFactory.bind(LoggerFactory.java:149)
	at org.slf4j.LoggerFactory.performInitialization(LoggerFactory.java:141)
	at org.slf4j.LoggerFactory.getProvider(LoggerFactory.java:419)
	at org.slf4j.LoggerFactory.getILoggerFactory(LoggerFactory.java:405)
	at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:354)
	at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:380)
	at edu.umd.cs.findbugs.Project.<clinit>(Project.java:94)
	at org.jetbrains.plugins.spotbugs.core.FindBugsProjects.lambda$get$0(FindBugsProjects.java:129)
	at org.jetbrains.plugins.spotbugs.common.util.WithPluginClassloader.notNull(WithPluginClassloader.java:58)
	at org.jetbrains.plugins.spotbugs.core.FindBugsProjects.get(FindBugsProjects.java:129)
	at org.jetbrains.plugins.spotbugs.core.FindBugsProjects.addFile(FindBugsProjects.java:119)
	at org.jetbrains.plugins.spotbugs.core.FindBugsProjects.addFiles(FindBugsProjects.java:68)
	at org.jetbrains.plugins.spotbugs.core.CheckinHandlerFactoryImpl$1$2.configure(CheckinHandlerFactoryImpl.java:101)
	at org.jetbrains.plugins.spotbugs.core.FindBugsStarter.lambda$asyncStartImpl$3(FindBugsStarter.java:202)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:894)
	at org.jetbrains.plugins.spotbugs.core.FindBugsStarter.asyncStartImpl(FindBugsStarter.java:201)
	at org.jetbrains.plugins.spotbugs.core.FindBugsStarter.asyncStart(FindBugsStarter.java:191)
	at org.jetbrains.plugins.spotbugs.core.FindBugsStarter.access$000(FindBugsStarter.java:54)
	at org.jetbrains.plugins.spotbugs.core.FindBugsStarter$1.run(FindBugsStarter.java:168)
	at com.intellij.openapi.progress.impl.CoreProgressManager.startTask(CoreProgressManager.java:434)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.startTask(ProgressManagerImpl.java:132)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressSynchronously$8(CoreProgressManager.java:518)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$new$0(ProgressRunner.java:84)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:248)
	at com.intellij.openapi.progress.ProgressManager.lambda$runProcess$0(ProgressManager.java:73)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$1(CoreProgressManager.java:192)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$12(CoreProgressManager.java:610)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:685)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:641)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:609)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:78)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:179)
	at com.intellij.openapi.progress.ProgressManager.runProcess(ProgressManager.java:73)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$5(ProgressRunner.java:248)
	at com.intellij.openapi.progress.impl.ProgressRunner$ProgressRunnable.run(ProgressRunner.java:498)
	at com.intellij.util.concurrency.ChildContext$runAsCoroutine$1.invoke(propagation.kt:81)
	at com.intellij.util.concurrency.ChildContext$runAsCoroutine$1.invoke(propagation.kt:81)
	at com.intellij.util.concurrency.ChildContext.runAsCoroutine(propagation.kt:86)
	at com.intellij.util.concurrency.ChildContext.runAsCoroutine(propagation.kt:81)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$launchTask$18(ProgressRunner.java:464)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:702)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:699)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:699)
	at java.base/java.lang.Thread.run(Thread.java:840)

**Comment by NoeJuza** [comment]
Same issue this morning.
Env:
```
OS: EndeavourOS Linux x86_64
Host: OMEN by HP Laptop 16-c0xxx
Kernel: 6.1.66-1-lts
```
Intellij Version:
```
2023.3
```

**Comment by NoeJuza** [comment]
As a temporary fix, downgrading to IntelliJ version `2022.2.5` via jetbrain toolbox seems to work
